rts/Lua/LuaUnsyncedCtrl.cpp

```diff
@@ -4864,3 +4867,57 @@ int LuaUnsyncedCtrl::Yield(lua_State* L)
 	lua_pushboolean(L, true); //hint Lua should keep calling Yield
 	return 1;
 }
+
+/* NB: strings here are never cleaned up, but the use case assumes
+ * that they live a long time and there's just a handful of them */
+std::set <std::string> tracyLuaPlots;
+
+/*** Initialize a plot in Tracy for use in debugging or profiling
+ *
+ * @function Spring.LuaTracyPlotConfig
+ * @string plotName which should be initialized
+ * @string[opt] plotFormatType "Number"|"Percentage"|"Memory", default "Number"
+ * @bool[opt] step stepwise chart, default true is stepwise
+ * @bool[opt] fill color fill, default false is no fill
+ * @number[opt] color unit32 number as BGR color, default white
+ * @treturn nil
+ */
+
+int LuaUnsyncedCtrl::LuaTracyPlotConfig(lua_State* L)
+{
+    const auto plotName 			= luaL_checkstring(L, 1);
```

Comment:
whitespace seems to be spaces all across both functions
```suggestion
	const auto plotName 			= luaL_checkstring(L, 1);
```